Tests/XcodeGenKitTests/ProjectGeneratorTests.swift

```diff
@@ -991,7 +991,7 @@ class ProjectGeneratorTests: XCTestCase {
                 let project = Project(name: "test", targets: [app], packages: [
                     "XcodeGen": SwiftPackage(url: "http://github.com/yonaskolb/XcodeGen", versionRequirement: .branch("master")),
                     "Codability": SwiftPackage(url: "http://github.com/yonaskolb/Codability", versionRequirement: .exact("1.0.0")),
-                ], localPackages: ["../XcodeGen"], options: .init(localPackagesGroup: "MyPackages"))
+                ], localPackages: ["../XcodeGen" : LocalSwiftPackage(path: "../XcodeGen")], options: .init(localPackagesGroup: "MyPackages"))
```

Comment:
Can we use absolute paths?

Reply:
No, I found that we will meet not linking local package issue to project when using absolute paths.
This looks old bug and I fixed in latest commit, so please check it after pushed

Thank you for pointing out, I did not notice it! 👍